StreamChatClient_v3/APIClient/Endpoints/Payloads/GuestUserTokenPayload_Tests.swift

```diff
@@ -9,11 +9,11 @@ final class GuestUserTokenPayload_Tests: XCTestCase {
     let guestUserDefaultExtraDataJSON = XCTestCase.mockData(fromFile: "GuestUser+DefaultExtraData")
     let guestUserNoExtraDataJSON = XCTestCase.mockData(fromFile: "GuestUser+NoExtraData")
     let guestUserCustomExtraDataJSON = XCTestCase.mockData(fromFile: "GuestUser+CustomExtraData")
-
+    
     func test_guestUserDefaultExtraData_isSerialized() throws {
         let payload = try JSONDecoder.default.decode(GuestUserTokenPayload<NameAndImageExtraData>.self,
                                                      from: guestUserDefaultExtraDataJSON)
-
+        
```

Comment:
According to changes related to empty lines, it seems we have different **Xcode -> Text Editing -> Editing -> While Editing** settings. I guess you have `including white-space only lines` setting unchecked, right?

Reply:
The formatter should be able to take care of this. cc @buh ?